src/io/contsetch.c

```diff
@@ -21,3 +21,35 @@ s32 osContSetCh(u8 ch) {
     __osSiRelAccess();
     return ret;
 }
+
+/*
+ * Use a mask to detect controller channels instead, allowing the PIF to skip channels instead of
+ * spending time reading data.
+ * This does not work on iQue Player.
+ */
+s32 osContSetMask(u8 ch) {
+    s32 ret = 0;
+
+#ifdef BBPLAYER
+    return ret;
```

Comment:
I think for iQue, `__osControllerMask` should be set to all 4 controllers enabled and `__osMaxControllers` should be 
set to `MAXCONTROLLERS`